posts/understand-entando-identity-management-using-keycloak.md

```diff
@@ -0,0 +1,55 @@
+---
+authors:
+  - Ryan Walker
+tags:
+  - Entando
+  - Keycloak
+date: 2023-01-10T12:21:50.000Z
+title: "Understanding Entando Identity Management using Keycloak"
+---
+
+# What is Identity Management, and why do we need it?
+
+Identity and access management is a framework of policies and technologies to ensure authorized users have the appropriate access to technology resources to perform their job functions. It includes resources and policies encompassing an organization-wide process to correctly identify, authenticate, and authorize entities through user access rights and restrictions. Those entities can be users, groups of users, or software applications. Hardware and software resources are securely protected from unauthorized access thanks to identity management solutions.
+
+# Understanding Entando Identity Management
+
+The Keycloak service powers all authentication and authorization for Entando. Entando applications rely on a Keycloak instance that is either externally installed or specific to an application.
+
+## Using Keycloak for Entando Authentication
+
+Entando implements Keycloak as a central authentication point to provide a unified view of identity. Keycloak acts as an abstraction layer to the underlying Identity Provider (IDP), allowing Entando to integrate into other IDPs without modifying the source. This schema increases portability, which is beneficial for configuring all authentication for an Entando application in one place.
```

Comment:
The last sentence is a little confusing to me. I'm not sure what "This schema" is referencing. Or is it just a typo and should be "scheme"?